pkgs/by-name/an/ani-skip/package.nix

```
@@ -0,0 +1,54 @@
+{
+  stdenvNoCC,
+  fetchFromGitHub,
+  makeWrapper,
+  gnugrep,
+  gnused,
+  curl,
+  fzf,
+  lib,
+}:
+stdenvNoCC.mkDerivation (final: {
```

```suggestion
stdenvNoCC.mkDerivation (finalAttrs: {
```

I am not sure if there are clear guidelines on how this should be named, but we usually use `finalAttrs`.